tests/test_client/tests.py

```diff
@@ -85,6 +87,83 @@ def test_post(self):
         self.assertEqual(response.templates[0].name, 'POST Template')
         self.assertContains(response, 'Data received')
 
+    def test_json_post(self):
+        """ POST some JSON data to a view """
```

Comment:
I think we prefer docstrings without leading and trailing spaces, i.e.
_"""POST JSON data to a view."""_.

Reply:
Is there a preference? Every doc string seems to have it's own formatting. I'm almost tempted to unify all of them. But that would be a very different ticket.